doc/api/tls.markdown

```diff
@@ -764,6 +764,8 @@ of written data and all required TLS negotiation.
 This instance implements a duplex [Stream][] interfaces.  It has all the
 common stream methods and events.
 
+Methods that return TLS connection meta data (e.g. `getPeerCertificate()`) will only return data while the connection is open.
```

Comment:
Please wrap at 80 columns.  Maybe you can turn `getPeerCertificate()` into a link?